src/silx/gui/plot/ImageView.py

```diff
@@ -821,15 +821,29 @@ def __updateProfileMenu(self):
         self.__profileMenu.addAction(profile.lineAction)
         self.__profileMenu.addAction(profile.clearAction)
 
+    def _formatValueToString(self, value):
+        try:
+            if isinstance(value, numpy.ndarray):
+                if len(value) == 4:
+                    return "RGBA: %i %i %i %i" % (value[0], value[1], value[2], value[3])
```

Comment:
RGBA can also be float in [0, 1], this formatting works for both int and float:

```suggestion
                    return "RGBA: %.3g, %.3g, %.3g, %.3g" % (value[0], value[1], value[2], value[3])
```

I also added `,` separator, to me it makes it easier to parse visually.